src/lib/auth.ts

```diff
@@ -56,22 +56,23 @@ export const DEVICE_API_KEY_PERMISSIONS = [
 	`resin.device.get?${matchesActor}`,
 	`resin.device.update?${matchesActor}`,
 	`resin.supervisor_release.get?should_manage__device/any(d:d/${matchesActor})`,
-	`resin.application.read?${ownsDevice} or depends_on__application/any(a:a/${ownsDevice})`,
+	`resin.application.read?${ownsDevice} or depends_on__application/any(da:da/${ownsDevice})`,
+	`resin.application_tag.read?application/any(a:a/${ownsDevice} or a/depends_on__application/any(da:da/${ownsDevice}))`,
 	`resin.device_config_variable.get?device/any(d:d/${matchesActor} or d/belongs_to__application/any(a:a/depends_on__application/any(da:da/${ownsDevice})))`,
 	`resin.device_config_variable.set?device/any(d:d/${matchesActor})`,
 	`resin.device_tag.get?device/any(d:d/${matchesActor})`,
 	`resin.device_tag.set?device/any(d:d/${matchesActor})`,
 	`resin.application_config_variable.get?${applicationControlsDevice}`,
-	`resin.release.read?should_be_running_on__device/any(d:d/${matchesActor})`,
-	`resin.release.read?${belongsToApplicationOrIsManagedByDevice}`,
+	`resin.release.read?should_be_running_on__device/any(d:d/${matchesActor}) or ${belongsToApplicationOrIsManagedByDevice}`,
+	`resin.release_tag.read?release/any(r:r/should_be_running_on__device/any(d:d/${matchesActor}) or r/${belongsToApplicationOrIsManagedByDevice})`,
 	`resin.device_environment_variable.get?device/any(d:d/belongs_to__application/any(a:a/depends_on__application/any(da:da/${ownsDevice})))`,
 	`resin.device_environment_variable.all?device/any(d:d/${matchesActor})`,
 	`resin.application_environment_variable.get?${applicationControlsDevice}`,
 
 	// Dependent device permissions
-	`resin.device.create?belongs_to__application/any(da:da/depends_on__application/any(a:a/${ownsDevice}))`,
-	`resin.device.read?belongs_to__application/any(da:da/depends_on__application/any(a:a/${ownsDevice}))`,
-	`resin.device.update?belongs_to__application/any(da:da/depends_on__application/any(a:a/${ownsDevice}))`,
+	`resin.device.create?belongs_to__application/any(a:a/depends_on__application/any(da:da/${ownsDevice}))`,
```

Comment:
I assume you've changed this because you think the second one is the dependent application? That's not the case though because you have `dependentApp depends on mainApp` so the aliases are now wrong/confusing

Reply:
I didn't put any thought on this. The reasons I changed the aliases are:

1. other permissions a few lines above (eg. the permission for `device_environment_variable`) use the `da` alias for `depends_on__application `.
2. reusing the same alias both for `belongs_to__application` and `depends_on__application` felt confusing to me.

So the point of this change was to get consistency among the permissions. Happy to change all to `a:a/...` instead.